### ISSUE TITLE
Cannot expand a complex type to more than 5K nodes

### ISSUE THREAD

**Issue by lothemar** [nikku/node-xsd-schema-validator#44]
### Describe the Bug
When parsing my schema, I get `"[fatal] Current configuration of the parser doesn't allow the expansion of a content model for a complex type to contain more than 5,000 nodes."`

### Expected Behavior
I should be able to parse my schema by editing a flag to allow more than 5k nodes

**Comment by nikku** [comment]
Agreed that this should be possible, but who uses a complex type with 5000 nodes? :see_no_evil:

**Comment by lothemar** [comment]
The European Commission's CBAM XSD definition for generated CBAM reports

Sadly not up to me :(

**Comment by VanguardaSistemas** [comment]
Hi! I'm facing the same problem.
Since @lothemar didn't send the files, i'm going to share the ones i had the problem.

I was unable to attach the files here, but i uploaded them here: https://limewire.com/?referrer=pq7i8xx7p2

There are 4 files:
- mdfe.xml (to validate against mdfe_v3.00.xsd)
- mdfe_v3.00.xsd
- mdfeTiposBasico_v3.00.xsd (that is included by mdfe_v3.00.xsd)
- tiposGeralMDFe_v3.00.xsd (that is included by mdfeTiposBasico_v3.00.xsd)

**Comment by nikku** [comment]
@VanguardaSistemas are these files liberally licensed, so you can include them in this repository as a test case?

If so, consider to PR a failing test case with these resources. We need to be able to reproduce the problem first, using publicly accessible schemata. Only then we can proceed to fix the issue.

**Comment by nikku** [comment]
Likely relevant for the discussion: https://docs.oracle.com/javase/7/docs/technotes/guides/xml/jaxp/JAXP-Compatibility_160.html#JAXP_security.

**Comment by VanguardaSistemas** [comment]
@nikku Yes, they are public, they are schemas for validating tax documents.
I've never done a PR, but I'll manage here

**Comment by nikku** [comment]
Fixed via https://github.com/nikku/node-xsd-schema-validator/pull/52, thanks @VanguardaSistemas for your PR with the test cases (https://github.com/nikku/node-xsd-schema-validator/pull/50).

Huge schemata can lead to resource exhaustion, because of that you have to explicitly enable insecure processing (disable some safeguards):

```javascript
const validator = require('xsd-schema-validator');

const { valid } = await validator.validateXML({ file: XML_FILE }, HUGE_SCHEMA, { insecure: true });
```
